### DISCUSSION TITLE
RPG/Storytelling como estratégia pra relações de conflito

### DISCUSSION THREAD

**Discussion by fititnt** [frontendbr/forum#548] [Discussão]
Um problema extremamente comum, que impede criatividade/inovação em empresas são as relações de trabalho: as pessoas não se entendem e/ou acreditam que os outros são inconsistentes e/ou vingativos. Porém as pessoas sempre deixam pistas.

Narrativa (contar história) não é apenas educativo para crianças ou _coisa pra vender produto para clientes_), [mas pode ser usado como forma de terapia e resolver conflitos dentro de uma empresa](https://en.wikipedia.org/wiki/Storytelling#In_business). Como é muito difícil ter "uma pessoa que saiba tudo", penso que seria melhor dividir essa tarefa, e minha proposta seria [fazer como jogo de interpretação de papeis](https://pt.wikipedia.org/wiki/Role-playing_game) temático, dramatizando um caso de uma pessoa começando a trabalhar em uma empresa **difícil** de lidar com os colegas, que descobre problemas graves nela e, suas ações, podem levar a crescer, ser boicotada ou mesmo ser demitida.

A ideia toda pode ter sua viabilidade discutida, porém imagino desde já que há, necessariamente, dois grupos:

- Do protagonista (que pode, na verdade, ser um grupo muito maior de pessoas, que decidem qual ação do protagonista; a discussão de como devem reagir pode até ser pública)
- Do restante da empresa que reage ao protagonista (aqui um grupo de mestres; sigilo sobre as ações)

_Nota: como o objetivo aqui não é ser um RPG "apelas lúdico" mas concentrar o máximo de esforço para o protagonista resolver situações de conflito dramáticas, acho que tende a ser melhor não ter mais atores na mesma história, **porém nada impede de se ter, ao mesmo tempo, mais de uma história ocorrendo**, e as pessoas tentando abordagens diferentes em cada uma delas_

## O Conto do Estagiário Que Sabia Demais

> Esse conto começa com você, que é estagiário em uma grande empresa, uma das maiores do seu país, e está feliz e contente com seu trabalho, e seu salário é aceitável, porém sua esperança é ser efetivado. Você até acha estranho como alguns estagiários anteriores chegaram a ser demitidos por erros absurdos e até mesmo, reza a lenda, alguém que anos atrás foi preso.
>
> Sua aventura (ou drama) começa minutos depois de receber uma ligação telefônica reclamando de um bug de usuário final, que é tão chocante que você simplesmente deixa o telefone cair no chão e fica sem resposta...

**Comment by fititnt** [comment]
Se fosse executado em apenas um dia, isso seria parecido com [psicodrama](https://pt.wikipedia.org/wiki/Psicodrama). Existem estilos diferentes de RPG quanto a duração, alguns duram por anos, outros que duram uma tarde.

Uma ideia aqui seria ver gente para fazer os dois times e usar situações bem hardcore e tensas, pra fazer as pessoas pensarem sobre dilemas morais e até sobre ações e reações. Se o pessoal for muito tímido, daria pra fazer apenas por hangout privado com algumas pessoas.

Outra opção (que tende a ser a que é mais interessante até pra quem companha no futuro) seria fazer os turnos a cada 2 ou 3 dias, senão mais tempo, para intencionalmente pra deixar os _dois lados_ pensando com mais calma, e não atrapalhar as relações de trabalho, e deixar em algum lugar no github.

Mesmo que não vire um "jogo publico", o simples fato de estimular esse tipo de pensamento aqui, já serve pra quando colegas forem tomar cerveja junto, ou então se reunir na cada dos outros, possam fazer "jogos mentais" assumindo cenários que podem ajudar nas relações de trabalho no futuro, sem precisar discutir problemas _mais simples_ e reais comparados aos que são fantasiosos.

**Comment by eduardojmatos** [comment]
@fititnt posso mudar o título pra: **"RPG/Storytelling como estratégia pra relações de conflito"**. Apenas pra diminuir o título da issue. :)

**Comment by nandomoreirame** [comment]
:thinking:

**Comment by bernardodiasc** [comment]
Há uma partida de RPG acontecendo no slack do X-Team, ja tem alguns meses, se não me engano estamos no 3o capítulo da história. Não sei se o objetivo disso foi resolução de conflito ou apenas um espaço pra descontrair, especialmente para aqueles que ja curtem RPG. Só sei que é muito divertido.

**Comment by fititnt** [comment]
Testei dramatização (papel vendedor/comprador) com amigo de infância que pensa em empreender, foco empresa dele em 2-5 anos vendendo para que atualmente é empregado.

## Antes disso eu tinha:
- Ele viu antes: Considerando lado da empresa: o que mudar na cultura dos desenvolvedores para aumentar competitividade do Brasil? #555
- Ele viu antes: Opiniões para melhorar relação do Brasil entre importação e exportação de tecnologia, foco software #557
- Falei em video conferência, audio e vídeo, por mais de uma hora e meia sobre meta-aprendizagem, [Hierarquia de necessidades de Maslow](https://pt.wikipedia.org/wiki/Hierarquia_de_necessidades_de_Maslow), e [flow/fluxo](https://en.wikipedia.org/wiki/Flow_(psychology)) como conceito para aumento de produtividade
- Falei bastande de [Zona de desenvolvimento proximal](https://pt.wikipedia.org/wiki/Zona_de_desenvolvimento_proximal) e básico de pedagogia; sobre porque mesmo que uma pessoa tiver ferramenta de graça, ela não será tão eficiente a ponto de usá-las; de que pedagogicamente falando é util ter apoio externo para ajudar a pessoa a encontrar seu caminho de ZDP, e que esse caminho é individual, não da para tratar todo mundo igual

## O que eu fiz:
- Eu reusei toda argumentação dele sobre o que ele falava do próprio ramo de atuação, e fiz papel do que poderia ser ele mesmo em 2 a 5 anos
- Eu não sabia os números exatos; [mas fiz inferências](https://pt.wikipedia.org/wiki/Infer%C3%AAncia) com outras áreas; ou seja, eu não precisava estar certo, só estava interpretando papeis por uma entrevista que durou uns 10 minutos até fechar o negócio
- Eu "convenci" a comprar.  Na verdade, fiz uma dramatização do que pode ter sido usado numa reunião que tal colega não estava) onde eu dizia que cobraria algo como _10x para vender um projeto sem restrição de uso de hardware, porém 4x + venda casada contratual para além de projeto de peças, vender as peças eu mesmo pela minha rede de contatos_

## O que ele respondeu logo depois da dramatização
- "Olha, isso é bom para eu ver como é lidar com convencer pessoas"

## O que fizemos depois da dramatização
Voltamos a pensar como tornar a minha dramatização (que no caso, seria ele) viável no menor tempo possível. Eu fiquei surpreso, porque ao pensar junto com colega de forma global, levando não só interesses de ganhar dinheiro imediato, a ideia de _gerar mais riqueza com menor esforço_ o empurra a trabalhar menos como especialista e mais como pessoa que une pontos. Porque a carência no Brasil de alta tecnologia, até mesmo no setor dele, vala a pena fazer parcerias internacionais.


- A dramatização, e pensamento divergente, e ser mais _agressivo_, fez os dois envolvidos chegarem a conclusões que, basicamente, empresas grandes já fazem.
- Meu amigo citou joint venture internacionais
- Reforçamos a ideia de "melhor dinheiro aqui do que lá fora"
- De forma não surpreendente, os valores absurdos de importação de hardware (peças, maquinário, etc) é extremamente interessante pra na pior das hipóteses, fazer parcerias internacionais e produzir no Brasil, não la fora e importar, e ter margens de lucro boas e até menos risco inicial comparado a desenvolver certos tipos de produto do zero
- Eu falei que era importante aprender sobre [metodos como Kaizen](https://pt.wikipedia.org/wiki/Kaizen) pois se empresas fossem compradas por japoneses, eles aplicariam sem dó e demitiriam pessoas se necessário; Meu amigo fez links com casos de empresas no passado que tiveram aumento de N vezes produtividade em chão de fabrica ao contratar especialista em otimização de processos que cobrava duzias de milhares por mes;

## Comentários gerais sobre interpretação de papeis como desencadeador de criatividade para emprededorismo

Essa dramatização não é exatamente sobre "como estratégia pra relações de conflito" mas creio que vale a pena por como exemplo aqui.

Toda essa história de pensar no Brasil como um todo, e incitar e ser incitado a gerar riqueza e considerar outros profissionais precisam de ajuda, e por isso pessoas como meu colega teriam que tomar a frente pra dar emprego,  em uma sessão de pouco mais de 3 horas parece levar mesmo a deduzir padrões que são usados por empresas.

Isto é, não foi só meu colega que aprendeu. Esse uso teatral, com preparação e depois discussão parece muito eficiente para incentivar criatividade, e também força de vontade de sentir que faz parte de algo maior a longo prazo vai de encontro a reforçar motivações.

Pretendo fazer isso outras vezes, preferencialmente com pessoas diferentes áreas de conhecimento, tentando dramatizar como interpretar cenas e resolver problemas. No nosso caso tinha eu de full stack developer e meu colega projetista mecânico, mas eu senti falta até de advogado (ou pessoa com conhecimento em comércio exterior) e vendedor

**Comment by lee101** [comment]
AI ahora está haciendo esto con AI Narrado y AI Texto generado Aventuras/personajes: https://netwrck.com
